api-reference/beta/api/connectorgroup_post_members.md

```diff
@@ -1,6 +1,6 @@
-# Add connector to connectorGroup
+# Add connector to connectorGroups
```

Comment:
The originals are all correct.  The title could be changed to Add a connector to a connectorGroup.